sys/mips/timer.c

```diff
@@ -117,7 +135,6 @@ static int mips_timer_attach(device_t *dev) {
   };
 
   tm_register(&state->timer);
-  tm_select(&state->timer);
```

Comment:
why now we don't need to set `time_source`? is the `time_source` used anywhere?

Reply:
It is set later in `init_clock. It is used for time access.